KeeTrayTOTP/TrayTOTP_Plugin.cs

```diff
@@ -74,7 +74,7 @@ public sealed partial class KeeTrayTOTPExt : Plugin
         /// </summary>
         internal const int setstat_int_EntryList_RefreshRate = 300;
         internal const int setstat_trim_text_length = 25;
-        public IReadOnlyList<string> setstat_allowed_lengths = new ReadOnlyCollection<string>(new[] { "6", "7", "8", "S" });
+        internal IReadOnlyList<string> setstat_allowed_lengths = new ReadOnlyCollection<string>(new[] { "6", "7", "8", "S" });
```

Comment:
Just a suggestion: This one could be private, but I see that the visibility in the rest of the files is mostly internal. I have some plans to refactor that when I have some more time on my hands.

Reply:
I would even suggest an enum for this, but I didn't want to make too much changes in one PR.